api/v1/eks_logforwarder_types.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2024 Tigera, Inc. All rights reserved.
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+)
+
+// EKSLogForwarderDeployment is the configuration for the EKSLogForwarder Deployment.
+type EKSLogForwarderDeployment struct {
```

Comment:
I realize now that this feature and this deployment are already exposed in the API under the name `EksCloudwatchLog`. Therefore, I think we should rename `EKSLogForwarderDeployment` to `EksCloudwatchLogDeployment` and we should   ideally also change the deployment and container names to reflect this in order to have the most consistent API terminology.